inst/schema/ModelResultResponse.schema.json

```diff
@@ -6,7 +6,13 @@
     "complete": {
       "type": "boolean",
       "enum": [ true ]
+    },
+    "warnings": {
```

Comment:
I suppose we could actually factor out this shared block into a single schema `Warnings.schema.json`